examples/phi3v_example.py

```diff
@@ -19,11 +19,7 @@ def run_phi3v():
     llm = LLM(
         model=model_path,
         trust_remote_code=True,
-        image_token_id=32044,
-        image_input_shape="1,3,1008,1344",
-        # Use the maximum possible value for memory profiling
-        image_feature_size=2653,
-        max_num_seqs=5,
+        max_num_seqs=1,
```

Comment:
setting to 5 gives me this:
```
[rank0]: Traceback (most recent call last):
[rank0]:   File "/home/ray/default/vllm/examples/phi3v_example.py", line 60, in <module>
[rank0]:     run_phi3v()
[rank0]:   File "/home/ray/default/vllm/examples/phi3v_example.py", line 19, in run_phi3v
[rank0]:     llm = LLM(
[rank0]:   File "/home/ray/default/vllm/vllm/entrypoints/llm.py", line 150, in __init__
[rank0]:     self.llm_engine = LLMEngine.from_engine_args(
[rank0]:   File "/home/ray/default/vllm/vllm/engine/llm_engine.py", line 413, in from_engine_args
[rank0]:     engine = cls(
[rank0]:   File "/home/ray/default/vllm/vllm/engine/llm_engine.py", line 255, in __init__
[rank0]:     self._initialize_kv_caches()
[rank0]:   File "/home/ray/default/vllm/vllm/engine/llm_engine.py", line 365, in _initialize_kv_caches
[rank0]:     self.model_executor.initialize_cache(num_gpu_blocks, num_cpu_blocks)
[rank0]:   File "/home/ray/default/vllm/vllm/executor/gpu_executor.py", line 87, in initialize_cache
[rank0]:     self.driver_worker.initialize_cache(num_gpu_blocks, num_cpu_blocks)
[rank0]:   File "/home/ray/default/vllm/vllm/worker/worker.py", line 205, in initialize_cache
[rank0]:     raise_if_cache_size_invalid(num_gpu_blocks,
[rank0]:   File "/home/ray/default/vllm/vllm/worker/worker.py", line 347, in raise_if_cache_size_invalid
[rank0]:     raise ValueError("No available memory for the cache blocks. "
[rank0]: ValueError: No available memory for the cache blocks. Try increasing `gpu_memory_utilization` when initializing the engine.
```

Reply:
Running on A10

Reply:
Nope. Just tried.
Even 1 gives me this
```
[rank0]: ValueError: The model's max seq len (131072) is larger than the maximum number of tokens that can be stored in KV cache (368). Try increasing `gpu_memory_utilization` or decreasing `max_model_len` when initializing the engine.
```
Is A10 already considered too small?
For manual testing, I just commented out the code that checks max_model_len against max seq len.

Reply:
368 is incredibly small!

Reply:
I think you may have to reduce the context length `max_model_len` for your GPU. Nevertheless, 368 still looks quite odd. Can you double check the `seq_len` calculation?

Reply:
Ok checking out the current master to try out the same thing.

5:
```
[rank0]: Traceback (most recent call last):
[rank0]:   File "/home/ray/default/vllm/examples/phi3v_example.py", line 64, in <module>
[rank0]:     run_phi3v()
[rank0]:   File "/home/ray/default/vllm/examples/phi3v_example.py", line 19, in run_phi3v
[rank0]:     llm = LLM(
[rank0]:   File "/home/ray/default/vllm/vllm/entrypoints/llm.py", line 144, in __init__
[rank0]:     self.llm_engine = LLMEngine.from_engine_args(
[rank0]:   File "/home/ray/default/vllm/vllm/engine/llm_engine.py", line 414, in from_engine_args
[rank0]:     engine = cls(
[rank0]:   File "/home/ray/default/vllm/vllm/engine/llm_engine.py", line 256, in __init__
[rank0]:     self._initialize_kv_caches()
[rank0]:   File "/home/ray/default/vllm/vllm/engine/llm_engine.py", line 366, in _initialize_kv_caches
[rank0]:     self.model_executor.initialize_cache(num_gpu_blocks, num_cpu_blocks)
[rank0]:   File "/home/ray/default/vllm/vllm/executor/gpu_executor.py", line 87, in initialize_cache
[rank0]:     self.driver_worker.initialize_cache(num_gpu_blocks, num_cpu_blocks)
[rank0]:   File "/home/ray/default/vllm/vllm/worker/worker.py", line 208, in initialize_cache
[rank0]:     raise_if_cache_size_invalid(num_gpu_blocks,
[rank0]:   File "/home/ray/default/vllm/vllm/worker/worker.py", line 350, in raise_if_cache_size_invalid
[rank0]:     raise ValueError("No available memory for the cache blocks. "
[rank0]: ValueError: No available memory for the cache blocks. Try increasing `gpu_memory_utilization` when initializing the engine.
```

2: same

1
```
[rank0]: Traceback (most recent call last):
[rank0]:   File "/home/ray/default/vllm/examples/phi3v_example.py", line 64, in <module>
[rank0]:     run_phi3v()
[rank0]:   File "/home/ray/default/vllm/examples/phi3v_example.py", line 19, in run_phi3v
[rank0]:     llm = LLM(
[rank0]:   File "/home/ray/default/vllm/vllm/entrypoints/llm.py", line 144, in __init__
[rank0]:     self.llm_engine = LLMEngine.from_engine_args(
[rank0]:   File "/home/ray/default/vllm/vllm/engine/llm_engine.py", line 414, in from_engine_args
[rank0]:     engine = cls(
[rank0]:   File "/home/ray/default/vllm/vllm/engine/llm_engine.py", line 256, in __init__
[rank0]:     self._initialize_kv_caches()
[rank0]:   File "/home/ray/default/vllm/vllm/engine/llm_engine.py", line 366, in _initialize_kv_caches
[rank0]:     self.model_executor.initialize_cache(num_gpu_blocks, num_cpu_blocks)
[rank0]:   File "/home/ray/default/vllm/vllm/executor/gpu_executor.py", line 87, in initialize_cache
[rank0]:     self.driver_worker.initialize_cache(num_gpu_blocks, num_cpu_blocks)
[rank0]:   File "/home/ray/default/vllm/vllm/worker/worker.py", line 208, in initialize_cache
[rank0]:     raise_if_cache_size_invalid(num_gpu_blocks,
[rank0]:   File "/home/ray/default/vllm/vllm/worker/worker.py", line 357, in raise_if_cache_size_invalid
[rank0]:     raise ValueError(
[rank0]: ValueError: The model's max seq len (131072) is larger than the maximum number of tokens that can be stored in KV cache (368). Try increasing `gpu_memory_utilization` or decreasing `max_model_len` when initializing the engine.
```

Reply:
368 is indeed too small. 
also this model in particular runs way too slow than other models that I tried regularly like Llava series.
Considering this model is compact in size.

Reply:
Then I think it's indeed due to long context length. Rather than changing the example's `max_num_seqs` (basically number of batches), you should reduce the `max_model_len`.